esercizio_doppio_array.c

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file esercizio_doppio_array.c
+ * @description Esercizio di inserimento dati in un Array, copia in ordine opposto in un altro Array i valori e stamparli entrambi.
+ * @author Lombardi Michele 
+ */
+
+
+/*
+OUTPUT DEL PROGRAMMA
+----------------------------------------------------------------------
+
+*/
+
+
+
+#include <stdio.h> // Libreria standard IO 
+#define N 3 //define di dimensione array
+
+int main(void)
+{
+    int v[N], v2[N], i2=0;

```

Comment:
Commenta liena per liena.. esempio, qui //allocazione di due array di dimensione fissa N e una variabile int a vlaore iniziale 0
```suggestion
    int v[N], v2[N], i2=0;
```